rules/nette-code-quality/tests/Rector/Identical/SubstrMinusToStringEndsWithRector/Fixture/with_var_value.php.inc

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rector\NetteCodeQuality\Tests\Rector\Identical\SubstrMinusToStringEndsWithRector\Fixture;
+
+function withVarValue($var, $value)
+{
+    substr($var, -4) !== $value;
+    $value !== substr($var, -4);
+    substr($var, -4) === $value;
+    $value === substr($var, -4);
```

Comment:
There should be a skipped test fixture with different number than word length.




```php
'part' !== substr($var, -3);

Reply:
implemented if comparison with string, check length of string first.